OrbitGl/ThreadTrack.cpp

```diff
@@ -229,8 +235,11 @@ float ThreadTrack::GetHeight() const {
   const uint32_t collapsed_depth = (GetNumTimers() == 0) ? 0 : 1;
   const uint32_t depth = is_collapsed ? collapsed_depth : GetDepth();
   return layout.GetTextBoxHeight() * depth +
-         (depth > 0 ? layout.GetSpaceBetweenTracksAndThread() : 0) + layout.GetEventTrackHeight() +
-         layout.GetTrackBottomMargin() + tracepoint_track_->GetHeight();
+         (depth > 0 ? layout.GetSpaceBetweenTracksAndThread() : 0) +
+         (thread_id_ != SamplingProfiler::kAllTracepointsFakeTid
+              ? layout.GetEventTrackHeight() + layout.GetTrackBottomMargin()
+              : 0) +
+         tracepoint_track_->GetHeight();
```

Comment:
Same as above, it would be better to avoid introducing a dependency to SamplingProfiler.